rails-snow-queen/app/assets/javascripts/quote.js

```diff
@@ -0,0 +1,60 @@
+class Quote {
+  constructor(config) {
+    this.errors = null;
+    this.onOpen = config.onOpen;
+    
+    this.handleErrors = this.handleErrors.bind(this);
+    this.handleOpenModal = this.handleOpenModal.bind(this);
+    this.addListeners();
+  }
+  
+  addListeners() {
+    $("#submitQuoteModal").on("show.bs.modal", this.handleOpenModal);
+    document.getElementById("new_quote").addEventListener("ajax:error", this.handleErrors);
+    document.getElementById("new_quote").addEventListener("ajax:success", () => { $("#submitQuoteModal").modal("hide") });
```

Comment:
Shouldn't this be also showing the `successModal`? Suggest moving this to a `this.handleSuccess`

Reply:
I removed the handleSuccess because it was just one line, not a good practice?

Reply:
done